plugins/woocommerce/tests/api-core-tests/playwright.config.js

```diff
@@ -53,7 +53,7 @@ const config = {
 		video: 'on-first-retry',
 		trace: 'retain-on-failure',
 		viewport: { width: 1280, height: 720 },
-		baseURL,
+		baseURL:baseURL,
```

Comment:
Minor nitpick, but let's add a space here for readability:

```suggestion
		baseURL: baseURL,
```

Reply:
I like it 👍